README.md

```diff
@@ -4,15 +4,45 @@ The samples in this repository demonstrate the various capabilities of the [Temp
 
 If you want to learn more about the Temporal Server and how it works, read the documentation at [https://docs.temporal.io](https://docs.temporal.io).
 
```

Comment:
May call out the difference between Docker Compose setup and Local setup here -

Reply:
I can add more info tomorrow unless you'll do it.